### PR TITLE
Add Ukrainian translations for the whole project

### PR DESCRIPTION
**Pull request by richardgrey**
#### Description of what you did:
* UK locale added across the project.
* Minor misspellings in English translations.
* Few missing translations.
* Fix for input placeholders in email-templates forms at users-permissions plugin (proper ids and removing HTML tags from translation, as react-intl doesn't allow them).

### CODE REVIEW

File: packages/strapi-plugin-email/admin/src/translations/uk.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "ConfigPage.description": "Налаштуйте плагін email Configure the email plugin",
+  "ConfigPage.title": "Email - Налаштування",
+  "EditForm.Input.number.label": "Максимально дозволений розмір (MB)",
+  "EditForm.Input.select.inputDescription": "Листи можуть відправлятися провайдером за замовчуванням (Sendmail) або зовнішніми провайдерами",
+  "EditForm.Input.select.label": "Провайдери",
+  "EditForm.Input.toggle.label": "Увімкнути відправку листів",
+  "notification.config.success": "Налаштування оновлено",
+  "plugin.description.long": "Надсилайте листи електронною поштою.",
```
Comment (soupette) [inline packages/strapi-plugin-email/admin/src/translations/uk.json:9]: Only the "plugin.description.long" & "plugin.description.short" are needed for this plugin. Can you remove the other ones?

File: packages/strapi-plugin-email/admin/src/translations/en.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "EditForm.Input.select.inputDescription": "Emails can be sent with the default provider (Sendmail) or an external provider",
   "EditForm.Input.select.label": "Providers",
   "EditForm.Input.toggle.label": "Enable email send",
-  "notification.config.success": "The settings has been updated",
+  "notification.config.success": "The settings have been updated",
```
Comment (soupette) [inline packages/strapi-plugin-email/admin/src/translations/en.json:8]: You have a conflict on this file can you remove all the translation except the two last one?

File: packages/strapi-plugin-users-permissions/admin/src/components/ListRow/index.js [hunk 3]
```diff
@@ -93,9 +94,13 @@ class ListRow extends React.Component {
             </div>
             <div className="col-md-6" style={{ fontWeight: '500' }}>
               {get(this.props.values, [get(this.props.item, 'name'), 'enabled']) ? (
-                <span style={{ color: '#5A9E06' }}>Enabled</span>
+                <span style={{ color: '#5A9E06' }}>
+                  <FormattedMessage id={getTrad('ListRow.enabled')} />
```
Comment (soupette) [inline packages/strapi-plugin-users-permissions/admin/src/components/ListRow/index.js:98]: This a lot for adding this trad!

File: packages/strapi-plugin-email/admin/src/translations/uk.json (hunk 1 above)
Comment (richardgrey) [reply packages/strapi-plugin-email/admin/src/translations/uk.json:2]: Thank you for pointing that. I've resolved the conflicts and clean up this file as well.

File: packages/strapi-plugin-email/admin/src/translations/en.json (hunk 2 above)
Comment (richardgrey) [reply packages/strapi-plugin-email/admin/src/translations/en.json]: Thank you, I've updated this branch to the latest master.

**Comment by alexandrebodin** [comment]
Hi @richardgrey can you resolve the conflicts in your PR please :)

**Comment by burlakko** [comment]
@alexandrebodin @richardgrey 
I'd like to help with this one. But don't really understand how a bunch of translation files may cause a `Knex: Timeout acquiring a connection` error.
Can you please guide me?

**Comment by alexandrebodin** [comment]
@richardgrey I'm not talking about the tests but the git conflicts :)

**Comment by richardgrey** [comment]
Hey @alexandrebodin! I've just updated the branch. Thank you for patience =)

**Pull request merged**